django/db/models/functions.py

```diff
@@ -137,3 +137,16 @@ class Upper(Func):
 
     def __init__(self, expression, **extra):
         super(Upper, self).__init__(expression, **extra)
+
+
+class Now(Func):
+    template = 'CURRENT_TIMESTAMP()'
```

Comment:
This will require an `as_oracle()` with `self.template = 'CURRENT_TIMESTAMP'` on the other end.